.circleci/config.yml

```diff
@@ -1,57 +1,44 @@
 version: 2.1
 
-commands:
-  package-install-test:
-    description: "install library from source distribution and run tests"
-    steps:
-      - checkout
-      - run : |
-          virtualenv venv
-          source venv/bin/activate
-          python setup.py sdist
-          LIBRARY_VERSION="featuretools_tsfresh_primitives-$(python setup.py --version)"
-          tar -zxvf "dist/${LIBRARY_VERSION}.tar.gz"
-          pip install -e "${LIBRARY_VERSION}/"
-          pip install -r "${LIBRARY_VERSION}/test-requirements.txt"
-      - run: |
-          source venv/bin/activate
-          cd "featuretools_tsfresh_primitives-$(python setup.py --version)/"
-          pytest
-
 jobs:
-  "python-27":
-    working_directory: ~/featuretools_tsfresh_primitives
+  test_python:
+    parameters:
+      python_version:
+        type: string
+        default: "3.7"
+    working_directory: ~/featuretools-tsfresh-primitives
     docker:
-      - image: circleci/python:2.7
+        - image: circleci/python:<< parameters.python_version >>
     steps:
-      - package-install-test
-
-  "python-35":
-    working_directory: ~/featuretools_tsfresh_primitives
-    docker:
-      - image: circleci/python:3.5
-    steps:
-      - package-install-test
-
-  "python-36":
-    working_directory: ~/featuretools_tsfresh_primitives
-    docker:
-      - image: circleci/python:3.6
-    steps:
-      - package-install-test
-
-  "python-37":
-    working_directory: ~/featuretools_tsfresh_primitives
-    docker:
-      - image: circleci/python:3.7
-    steps:
-      - package-install-test
+      - checkout
+      - run:
+          name: "Prepare Python Environemnt"
+          command: |
+            virtualenv env -q
+            source env/bin/activate
+            pip install -r test-requirements.txt -q
+            pip install . -q
+      - run:
+          name: "Run Unit Tests"
+          command: |
+            source env/bin/activate
+            pytest --cache-clear --show-capture=stderr --disable-warnings -vv
```

Comment:
I don't think we want to disable warnings

Reply:
enabled warnings